aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -770,7 +766,7 @@ Tensor& _int_mm_out_cuda(const Tensor& self, const Tensor& mat2, Tensor& result)
 
   TORCH_CHECK(result.is_contiguous(), "Expected result to be contiguous.");
 
-#if (!defined(USE_ROCM) && defined(CUDA_VERSION) && (CUDA_VERSION >= 11070)) || (defined(USE_ROCM) && (ROCM_VERSION >= 60000))
+#if (!defined(USE_ROCM) && defined(CUDA_VERSION) && (CUDA_VERSION >= 11070)) || defined(USE_ROCM)
```

Comment:
nit: maybe we can remove the `!defined(USE_ROCM)`

Reply:
Good catch.  Let me make a pass through all code again looking for this anti-pattern.

Reply:
Fixed.  Made a pass through the code and it all adheres to https://pytorch.org/docs/stable/notes/hip.html#cuda-api-to-hip-api-mappings-in-c now.